parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -247,11 +248,21 @@ extractHyperkube() {
 
 installKubeletAndKubectl() {
     if [[ ! -f "/usr/local/bin/kubectl-${KUBERNETES_VERSION}" ]]; then
-        if [[ "$CONTAINER_RUNTIME" == "docker" ]]; then
-            extractHyperkube "docker"
+        if version_gte ${KUBERNETES_VERSION} 1.17; then  # don't use hyperkube
+            K8S_DOWNLOAD_URL="https://dl.k8s.io/v${KUBERNETES_VERSION}/kubernetes-node-linux-amd64.tar.gz"
+            K8S_TGZ_TMP=$(echo ${K8S_DOWNLOAD_URL} | cut -d "/" -f 5)
+            mkdir -p "${K8S_DOWNLOADS_DIR}"
+            retrycmd_get_tarball 120 5 "$K8S_DOWNLOADS_DIR/${K8S_TGZ_TMP}" ${K8S_DOWNLOAD_URL} || exit $ERR_K8S_DOWNLOAD_TIMEOUT
```

Comment:
is the pre-download on VHD a todo?

Reply:
/hold

Yes it is a `// TODO:`, thanks for reviewing and keeping me honest.

Reply:
Fixed now, PTAL.

Reply:
/hold cancel